web/src/components/overview/StorageSection.tsx

```diff
@@ -104,81 +37,71 @@ const Content = ({ children }) => (
 
 /**
  * Text explaining the storage proposal
- *
- * FIXME: this needs to be basically rewritten. See
- * https://github.com/openSUSE/agama/discussions/778#discussioncomment-7715244
- *
- * @param {object} props
- * @param {Proposal} props.proposal
  */
 export default function StorageSection() {
-  const availableDevices = useAvailableDevices();
-  const result = useProposalResult();
-
-  if (result === undefined) return;
+  const drives = useConfigDevices().filter((d) => d.name);
```

Comment:
NP: all drives should have a name.

Reply:
BTW,  at this moment `useConfigDevices` returns drives only, but the model will be extended with LVM, etc. Should we filter drives here to be sure everything is a drive?

Reply:
Yes. This only adapts the summary to the very same cases that are handled by the storage section. We are not working on LVM support yet. If LVM is used, both this part and the storage configuration page would be equally broken.

We could of course filter, just to be safe. But then the information would be wrong (well, not fully precise) when we start adding LVM. I think I prefer to add a TODO to make it obvious and let it explode so it's not overlooked.